engine/lite/components/request-response/src/main/scala/pl/touk/nussknacker/engine/lite/components/requestresponse/jsonschema/sources/JsonSchemaRequestResponseSourceFactory.scala

```diff
@@ -36,11 +41,12 @@ class JsonSchemaRequestResponseSourceFactory extends RequestResponseSourceFactor
 
   override def implementation(params: Map[String, Any],
                               dependencies: List[NodeDependencyValue],
-                              finalStateOpt: Option[Schema]): RequestResponseSource[Any] = {
-    val finalSchemaState = finalStateOpt.getOrElse(throw new IllegalStateException("Unexpected (not defined) final state determined during parameters validation"))
+                              finalStateOpt: Option[(Schema, Schema)]): RequestResponseSource[Any] = {
+    val inputSchemaState = finalStateOpt.map(_._1).getOrElse(throw new IllegalStateException("Unexpected (not defined) final state determined during parameters validation"))
```

Comment:
Please use:
```
val (inputSchema, outputSchema) = finalStateOpt.getOrElse(throw new IllegalStateException("Unexpected (not defined) final state determined during parameters validation")) 
```
instead of cryptic _1, _2 construction